ui/src/index.js

```diff
@@ -3,4 +3,12 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+import PDClient from '@pingcap-incubator/pd-client-js';
+
+// TODO
+const pdClient = new PDClient({
```

Comment:
You can put it in `src/utils/client/index.js`, change it to expose two APIs:

```js
export default {
  dashboard: dashboardClient,
  pd: pdClient,
};
```

Reply:
Ok, will fix it soon.